images/kubeflow-pipelines/configs/latest.frontend.apko.yaml

```
@@ -0,0 +1,19 @@
+contents:
+  packages:
+    - nodejs
+    - kubeflow-pipelines-frontend
+  
```

```suggestion
  - py3-ml-metadata
  
```

Maybe adding the dependency like this is worth a try?

---

Now that the `py3-ml-metadata` package is fixed, adding this `wget` in seems to make the CI Helm test pass.

That said, I think nodejs and wget should be dependencies in the main frontend package, so am making a PR against that in Wolfi.